nestkernel/nest_names.cpp

```diff
@@ -77,7 +77,6 @@ const Name beta( "beta" );
 const Name beta_Ca( "beta_Ca" );
 const Name box( "box" );
 const Name buffer_size( "buffer_size" );
-const Name buffer_size_secondary_events( "buffer_size_secondary_events" );
 const Name buffer_size_spike_data( "buffer_size_spike_data" );
```

Comment:
`buffer_size_secondary_events` must also be removed in `nest_names.h` I think this might be why Travis is currently failing :)

Reply:
Can we confirm that this is the reason for the failed checks?